pkgs/top-level/all-packages.nix

```diff
@@ -21268,7 +21268,10 @@ in
   # - Update the rev in ../os-specific/linux/kernel/linux-libre.nix to the latest one.
 
   # Intentionally lacks recurseIntoAttrs, as -rc kernels will quite likely break out-of-tree modules and cause failed Hydra builds.
-  linuxPackages_testing = linuxPackagesFor pkgs.linux_testing;
+  linuxPackages_testing =
+    if lib.versionAtLeast linux_latest.version pkgs.linux_testing.version
+    then linuxPackages_latest
+    else linuxPackagesFor pkgs.linux_testing;
```

Comment:
```suggestion
  linuxPackages_testing =
    if linux_latest.kernelOlder linux_testing.version
    then linuxPackages_latest
    else linuxPackagesFor linux_testing;
```

Reply:
@Atemu Nice suggestion, didn't know about that function! But I think it should be `linux_latest.kernelAtLeast` or the arguments need to be flipped.
```
nix-repl> if linux_latest.kernelAtLeast linux_testing.version then linux_latest.version else linux_testing.version
"5.13.3"

nix-repl> if linux_latest.kernelOlder linux_testing.version then linux_latest.version else linux_testing.version
"5.13-rc6"
```

Reply:
Ah yes, didn't actually test.